Tests/tests_DataStore_initialise.py

```diff
@@ -0,0 +1,66 @@
+import unittest
+
+from Store.DataStoreModule import DataStore
+from sqlalchemy import inspect
+
+
+class TestDataStoreInitialise(unittest.TestCase):
+    def test_sqlite_initialise(self):
+        """Test whether schemas created successfully on SQLite"""
+        data_store_sqlite = DataStore("", "", "", 0, ":memory:", db_type="sqlite")
+
+        # inspector makes it possible to load lists of schema, table, column
+        # information for the sqlalchemy engine
+        inspector = inspect(data_store_sqlite.engine)
```

Comment:
I like it

Reply:
Thanks. I believe it's the best way to reach metadata of databases